doc/os_list.json

```diff
@@ -0,0 +1,22 @@
+{
+   "os_list": [
+       {
+           "name": "AOS",
+           "description": "AOS for Raspberry Pi",
+           "icon": "https://icon.png",
+           "subitems": [
+               {
+                   "name": "AOS",
```

Comment:
We will have two images one for nvme another for USB flash. Both should be here,